RF24Network.h

```diff
@@ -934,8 +944,11 @@ class ESBNetwork
 
     radio_t& radio; /** Underlying radio driver, provides link/physical layers */
 
-    uint8_t frame_size;                                                                            /* The outgoing frame's total size including the header info. Ranges [8, MAX_PAYLOAD_SIZE] */
-    const static unsigned int max_frame_payload_size = MAX_FRAME_SIZE - sizeof(RF24NetworkHeader); /* always 24 bytes to compensate for the frame's header */
+    uint8_t frame_size; /* The outgoing frame's total size including the header info. Ranges [8, MAX_PAYLOAD_SIZE] */
+
+    uint8_t max_frame_size;
```

Comment:
I think we can do without this new `max_frame_size` member. Instead, we could just set `max_frame_payload_size` in each specialized `write()` definition, then in `main_write()` do
```cpp
max_frame_payload_size -= sizeof(RF24NetworkHeader)
```